core/application/application_common.cc

```diff
@@ -169,11 +169,12 @@ ErrorCode ApplicationCommon::DiagConfig(
     return ErrorCode::kError;
   }
   nlohmann::json data = nlohmann::json::parse(file);
-  uint16_t app_id_{};
+  uint16_t app_id{};
   if (data.contains("app_id")) {
-    if (data.at("app_id").is_number()) {
-      app_id_ = data.at("app_id");
-      diag_controller.Init(app_id_);
+    if (data.at("app_id").is_string()) {
+      app_id = data.at("app_id");
+      diag_controller.Init(app_id);
+      this->app_id_ = app_id;
```

Comment:
po co aż takie zabawy ?? 
```suggestion
      this->app_id_ = data.at("app_id");
      diag_controller.Init(this->app_id_);
```

Reply:
błąd w rebase